### PR TITLE
feat: [IOPID-1062] Change copy

### PR DESCRIPTION
**Pull request by Ladirico**
## Short description
Change copy like this [figma](https://www.figma.com/file/BEXvjQmBkMRRFiNcMgE8g3/Login-Veloce?node-id=2522%3A23953&mode=dev)

## List of changes proposed in this pull request
- change text

## How to test
Run the project as first onboarding (i suggest to uninstall and reinstall app)

### CODE REVIEW

**Comment by Ladirico** [comment]
hi @AliceAzzolini !
Can you please fix the english text?

**Review by AliceAzzolini — APPROVED** [review]

**Review by shadowsheep1 — APPROVED** [review]
LGTM

File: locales/en/index.yml [hunk 1]
```diff
@@ -838,7 +838,7 @@ onboarding:
       invalid: The code is not valid
       match: The code doesn’t match
     policy:
-      headerTitle: "For security reasons, the code must not contain:"
+      headerTitle: "Scegli un codice difficile da individuare e che non contenga:"
```
Comment (Ladirico) [inline locales/en/index.yml:841]: @AliceAzzolini i need the english text! Thanks you so much!

Comment (AliceAzzolini) [reply locales/en/index.yml:841]:
```suggestion
      headerTitle: Choose a code that's hard to guess and doesn't contain:
```

**Pull request merged**